stream-chat-android-client/src/main/java/io/getstream/chat/android/client/experimental/plugin/listeners/SendReactionListener.kt

```diff
@@ -0,0 +1,58 @@
+package io.getstream.chat.android.client.experimental.plugin.listeners
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.utils.Result
+import io.getstream.chat.android.core.ExperimentalStreamChatApi
+
+/**
+ * Listener for [io.getstream.chat.android.client.ChatClient.sendReaction] calls.
+ */
+@ExperimentalStreamChatApi
```

Comment:
What is the timeline for this API to become non exp?

Reply:
After removing ChatDomain